THIRD-PARTY-LICENSES.txt

```diff
@@ -1,14 +1,14 @@
 
-Lists of 411 third-party dependencies.
+Lists of 399 third-party dependencies.
```

Comment:
12 down, I see things like `dockstore-integration-testing` and `dropwizard-testing` which makes me think a lot of what I thought would be removed is staying around from a licenses POV due to that test scope. 

What is the difference in jar size (or maven shade plugin warning)?

Reply:
Also relevant to https://github.com/dockstore/dockstore/issues/5549

Reply:
A develop webservice is 159M, I think that drop is significant.